lib/bolt/outputter/json.rb

```diff
@@ -90,7 +90,7 @@ def print_puppetfile_result(success, puppetfile, moduledir)
       end
 
       def print_targets(options)
-        targets = options[:targets].map(&:name)
+        targets = options[:targets].map(&:safe_name)
```

Comment:
This definitely feels wrong since we otherwise stringify the target as the name.